service/svc.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+ROOTPROJECTPATH="$(
+    cd -- "$(dirname "$0")/.." >/dev/null 2>&1
+    pwd -P
+)"
+
+echo $ROOTPROJECTPATH
+
+function install {
+    path=$1
+    if [ -z "$1" ]; then
+        path="/lib/systemd/system"
+    fi
+
+    if [ -e "$path/threeal-bot.service" ]; then
+        echo "Service is already installed"
+        exit
+    fi
+
+    workdir=$ROOTPROJECTPATH
+
+    execstart="go run $workdir/cmd/backend/main.go"
+
+    echo "Installing service..."
+
+    sed -e "s@<user>@$USER@" -e "s@<group>@$USER@" -e "s@<workdir>@$workdir@" -e "s@<pathexists>@$workdir@" -e "s@<execstart>@$execstart@" service/threeal-bot.service >threeal-bot.service
+
+    mv threeal-bot.service $path
```

Comment:
It seems more simpler to directly using the output redirect from the `sed` command.
```
sed -e ... > $path/threeal-bot.service
```

Reply:
I'm using `mv` because maybe the user wants to rename the file. If you think this is not necessary, I will change it.